src/main/java/io/ebean/config/ClassLoadConfig.java

```diff
@@ -118,7 +118,9 @@ public ClassLoader getClassLoader() {
   /**
    * Wraps the preferred, caller and context class loaders.
    */
-  protected class ClassLoaderContext {
+  protected static class ClassLoaderContext {
```

Comment:
As stated by the comment: this class is an inner class. Making it static results in it not keeping an implicit reference to its enclosing class. Such a reference could prevent the enclosing class from being GC'd. So that situation is a potential memory leak.